src/zkbob/utils/ZkBobAccounting.sol

```diff
@@ -325,6 +345,22 @@ contract ZkBobAccounting {
         emit UpdateLimits(_tier, tl);
     }
 
+    // Tier is set as per the KYC Providers recomendation only in the case if no specific tier
+    // assigned to the user
+    function _getTierForUser(address _user) internal view returns (uint8) {
+        UserStats memory us = userStats[_user];
```

Comment:
Seems that we load `userStats` from storage twice, right before calling this function and here as well. I think this could be optimized.

Reply:
Yes, it could: https://github.com/zkBob/zkbob-contracts/pull/54/commits/62a721e883ebe38e6cb3f1c3cc82c9738f3c543e